tests/cypress/support/commands.js

```diff
@@ -421,3 +421,9 @@ Cypress.Commands.add('dropDraggedItemAfter', {prevSubject: true}, (subject) => {
         );
     });
 });
+
+Cypress.Commands.add('checkAndCloseAlert', (text) => {
```

Comment:
Can you add the definition to command.d.ts ?
Vscode doesn't want to autocomplete the new commands without it.